ui-lib/src/components/ProgressBar/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { memo } from "react";
+import styled from "styled-components";
+import { border, BorderProps, color, ColorProps, space, SpaceProps } from "styled-system";
+import CheckAlone from "@ui/assets/icons/CheckAloneMedium";
+import Text from "@components/Text";
+import Flex from "@components/Layout/Flex";
+
+export interface Props {
+  /**
+   * An array of labels that will determine the progress bar steps.
+   */
+  steps: string[];
+  /**
+   * Index of the active step, starting at zero and defaulting to 0 if omitted.
+   */
+  activeIndex?: number;
+}
+/**
+ * The state of a progress bar step.
+ */
+type StepState = "pending" | "current" | "completed";
+export type StepProps = {
+  /**
+   * State of the step.
+   */
+  state: StepState;
+  /**
+   * The label to display.
+   */
+  label: string;
+  /**
+   * If true, hides the left "separator" bar that bridges the gap between the wider separator and the item.
+   */
+  hideLeftSeparator: boolean;
+  /**
+   * The next step state, or undefined if the current step is the last one.
+   */
+  nextState?: StepState;
+};
+
+export const Item = {
+  Container: styled.div.attrs(_ => ({
+    mx: 2,
+  }))<ColorProps & BorderProps & SpaceProps>`
```

Comment:
```suggestion
  Container: styled.div.attrs({ mx: 2 })<ColorProps & BorderProps & SpaceProps>`
```

If I remember correctly, the .attrs method accepts an object.

Reply:
Good catch 🎣